zenoh_router/assets/monitors/zenoh_router_disconnected.json

```
@@ -0,0 +1,35 @@
+{
+  "version": 2,
+  "created_at": "2023-12-20",
+  "last_updated_at": "2024-01-15",
+  "title": "No active sessions",
+  "tags": [
+    "integration:zenoh-router"
+  ],
+  "description": "To have connectivity Zenoh router should have at least one active session. This monitor tracks the count of active sessions. Alerts can indicate a connectivity issue or that the host.",
```

```suggestion
  "description": "To have connectivity, Zenoh router should have at least one active session. This monitor tracks the count of active sessions. Alerts can indicate a connectivity issue or that the host.",
```
It seems like something is missing here -- like "that the host" should have more content after it?

---

Fixed. Thanks!